src/lib/containers/JiraIssueCollector.tsx

```diff
@@ -143,7 +143,7 @@ export function useJiraIssueCollector(config: UseJiraIssueCollectorConfig) {
       } else {
         console.warn('Fetching profile data to display Jira popup')
       }
-    } else {
+    } else if (show && !accessToken) {
```

Comment:
just noticed this was being logged if !show, i.e. not a 500 error. We should only log that we're not showing the Jira popup if (show && !accessToken)